src/index.js

```diff
@@ -0,0 +1,200 @@
+    // findCityLocation(cityElement.value);
+}
+
+// Function to decrease the temperature
+function decreaseTemperature() {
+    state.temperature--;
+    updateTemperatureDisplay(state.temperature);
+    // findCityLocation(cityElement.value);
+}
+
+// Function to reset city name & clear input field
+function resetCityName() {
+    cityElement.value = '';
+    cityInput();
+}
+
+// Function to update the temperature display, change the text color, and update the landscape
+function updateTemperatureDisplay(temperature) {
+    temperatureElement.textContent = `${temperature}°F`;
+
+    if (temperature >= 80) {
+        temperatureElement.style.color = 'red';
+        emojiElement.textContent = '☀️🏝️🥵🌞☀️☀️🏝️🥵🌞☀️☀️🏝️🥵🌞☀️☀️🏝️🥵🌞☀️☀️🏝️🥵🌞☀️☀️🏝️🥵🌞☀️🏝️🥵🌞';
+        landscapeImg.src = 'styles/sunny1.jpeg';
+        landscapeImg.alt = 'Sunny';
+    } else if (temperature >= 70 && temperature <= 79) {
+        temperatureElement.style.color = 'grey';
+        emojiElement.textContent = '🌞😶‍🌫️🌼🌷😎🍂🌤️🌞😶‍🌫️🌼🌷😎🍂🌤️🌞😶‍🌫️🌼🌷😎🍂🌤️🌞😶‍🌫️🌼🌷😎🍂🌤️';
+        landscapeImg.src = 'styles/cloudy.jpeg';
+        landscapeImg.alt = 'Cloudy';
+    } else if (temperature >= 60 && temperature <= 69) {
+        temperatureElement.style.color = 'indigo';
+        emojiElement.textContent = '☁️🫥🌦️🌧️🌤️🌧️☔️🌈☁️🫥🌦️🌧️🌤️🌧️☔️🌈☁️🫥🌦️🌧️🌤️🌧️☔️🌈☁️🫥🌦️🌧️🌤️🌧️☔️🌈';
+        landscapeImg.src = 'styles/rainy.jpeg';
+        landscapeImg.alt = 'Rainy';
+    } else if (temperature >= 50 && temperature <= 59) {
+        temperatureElement.style.color = 'orange';
+        emojiElement.textContent = '🌤️🍁🥮⛅️🍂😎🍃🌤️🍁🥮⛅️🍂😎🍃🌤️🍁🥮⛅️🍂😎🍃🌤️🍁🥮⛅️🍂😎🍃🌤️';
+        landscapeImg.src = 'styles/autumn.jpeg';
+        landscapeImg.alt = 'Autumn';
+    } else {
+        temperatureElement.style.color = 'teal';
+        emojiElement.textContent = '❄️❄️☃️☃️⛄️⛄️🤶🏾🥶🥶🥶⛄️⛄️⛄️❄️❄️❄️❄️🌨️🌨️🌨️🌨️☃️⛷️⛄️🤶🏾🥶❄️☃️🌨️☃️⛷️⛄️🤶🏾🥶❄️☃️';
+        landscapeImg.src = 'styles/snowy.jpeg';
+        landscapeImg.alt = 'Snowy';
+    }
+}
+
+//function to return city input
+function cityInput() {
+    let cityName = document.getElementById("city").value;
+    document.getElementById("cityChoice").innerHTML = cityName;
+}
+//function to update sky selection when there is a change
+function skySelection() {
+    var choice = document.getElementById("sky").value;
+    updateSkyDisplay();
+}
+
+function updateSkyDisplay() {
+    let sky = document.getElementById("sky").value
+
+    if (sky == "sunny") {
+        skyEmojiElement.textContent = '🌞😎☀️☀🌼😎🌞😎☀️☀🌼😎🌞😎☀️☀🌼😎🌞😎☀️☀🌼😎🌞😎☀️☀🌼😎🌞😎☀️';
+
+    } else if (sky === "cloudy") {
+        skyEmojiElement.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️🌤 ☁️ ☁️☁️☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️☁️☁️☁️☁️ ☁️ 🌤 ☁️☁️☁️☁️🌤';
+
+    } else if (sky === "rainy") {
+        skyEmojiElement.textContent = '🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️☔️🌈🌦️🌧️🌧️';
+        
+    } else if (sky === "snowy") {
+        skyEmojiElement.textContent = '❄️❄️☃️☃️⛄️⛄️🤶🏾🥶🥶🥶⛄️⛄️⛄️❄️❄️❄️❄️🌨️🌨️🌨️🌨️☃️⛷️⛄️🤶🏾🥶❄️☃️🌨️☃️⛷️⛄️🤶🏾🥶❄️☃️❄️';
+    }
+    
+}
```

Comment:
Awesome job everyone! You really ate this one up! Please feel free to reach out to me if you have any questions about the comments I left or if you want to discuss anything in greater detail! ⭐️